src/main/java/uk/nhs/hee/tis/revalidation/connection/service/ConnectedElasticSearchService.java

```diff
@@ -72,13 +77,15 @@ public ConnectionSummaryDto searchForPage(String searchQuery, Pageable pageable)
 
       LOG.debug("Query {}", fullQuery);
 
-      Page<ConnectedView> result = connectedElasticSearchRepository.search(fullQuery, pageable);
+      Page<CurrentConnectionsView> result = currentConnectionElasticSearchRepository
+          .search(fullQuery, pageable);
```

Comment:
The `search` method is decrecated, we can use @Query annotation to rewrite it as what we did in Recommendation service. But it's not very clear what the searchQuery is, so I prefer to leave it there until we redo the searchQuery & filters.